nginx-app/templates/default/redirect-ssl.conf.erb

```diff
@@ -1,5 +1,6 @@
 server {
-    listen 443;
+    listen 80;
```

Comment:
Make them listen on both ports so we don't have too many configs to dig through. Also certbot challenge needs port 80 to verify.